doc/modules/reference.rst

```diff
@@ -277,3 +277,31 @@ uses.
    clean
 
 
+.. _connectivity_ref:
+
+:mod:`nilearn.connectivity`: Functional Connectivity
+====================================================
+
+.. automodule:: nilearn.connectivity
+   :no-members:
+   :no-inherited-members:
+
+**Classes**:
+
+.. currentmodule:: nilearn.connectivity.connectivity_matrices
```

Comment:
For the doc to be generated, it is necessary to specify the whole path. As in `.. currentmodule:: nilearn.plotting.displays` for the `OrthoSlicer`.